ui/src/overridden/components/Menu/RightMenuItem.js

```diff
@@ -1,46 +1,55 @@
-import { FrontSiteRoutes } from '@inveniosoftware/react-invenio-app-ils';
+import {
+  FrontSiteRoutes,
+  getStaticPageByName,
+  invenioConfig,
+} from '@inveniosoftware/react-invenio-app-ils';
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { Dropdown, Menu, Responsive } from 'semantic-ui-react';
-import { config } from '../../../config';
 
-const dropdownEntries = (
-  <Dropdown.Menu>
-    <Dropdown.Item as="a" href="https://library.web.cern.ch/resources/remote">
-      External Online Resources
-    </Dropdown.Item>
-    <Dropdown.Divider />
-    <Dropdown.Item as="a" href="#">
-      F.A.Q.
-    </Dropdown.Item>
-    <Dropdown.Divider />
-    <Dropdown.Item as="a" href={`mailto:${config.libraryEmail}`}>
-      Ask a librarian
-    </Dropdown.Item>
-    <Dropdown.Divider />
-    <Dropdown.Item as={Link} to={FrontSiteRoutes.documentRequestForm}>
-      Request new literature
-    </Dropdown.Item>
-  </Dropdown.Menu>
-);
+const DropdownItems = () => {
+  return (
+    <>
+      <Dropdown.Item as="a" href="https://library.web.cern.ch/resources/remote">
```

Comment:
nit: same here to make it part of the config, domains change etc..

Reply:
See my comment above. A change of its value will require a code change + re-deployment anyway, at the moment.